custom_config_data/config.yaml

```diff
@@ -6,9 +6,10 @@ CONFIG_NAME: holesky
 # ---------------------------------------------------------------
 # `2**14` (= 16,384)
 MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
-# Sep-15-2023 13:55:00 +UTC
-MIN_GENESIS_TIME: 1694786100
-GENESIS_FORK_VERSION: 0x00017000
+# Sep-28-2023 11:55:00 +UTC
+# This is an invalid valid and should be updated when you create the genesis
+MIN_GENESIS_TIME: 1695902100
+GENESIS_FORK_VERSION: 0x01017000
```

Comment:
😱😱😱
then we will have to update deposit-cli for it.

@barnabasbusa 
Is there also a new `genesis_validators_root `?

/cc @carlbeek

Reply:
The `genesis_validators_root` is `0x9143aa7c615a7f7115e2b6aac319c03529df8242ae705fba9df39b79c59fa8b1`